extensions/wikia/AdEngine/js/AdTracker.js

```diff
@@ -73,6 +78,32 @@ define('ext.wikia.adEngine.adTracker', ['ext.wikia.adEngine.utils.timeBuckets','
 	}
 
 	/**
+	 * A generic function to track an ad-related event and its timing in DataWarehouse
+	 *
+	 * @param {object} data - data to track as JS object (will be converted to URL-like query-string)
+	 */
+	function trackDW(data, eventName) {
+		var trackValue = {
+			eventName: eventName,
+			trackingMethod: 'internal'
+		};
+
+		if (typeof data === 'string') {
+			trackValue.ping = data;
+		} else {
+			//$.extend(trackValue, data);
+			Object.keys(data).forEach(function (k,v) {
+				trackValue[k] = v;
+			});
+		}
+		console.log("AAAAAA !!!!!!!");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)